vulnerability_alerts.exs

```diff
@@ -28,7 +27,7 @@ defmodule VulnerabilityAlerts do
     {
       repository(owner: "#{owner}", name: "#{name}") {
         nameWithOwner
-        vulnerabilityAlerts(first: 100) {
+        vulnerabilityAlerts(first: 100, states: OPEN) {
```

Comment:
Does it have to be in quotes or is it working like this directly?

Reply:
Tried it against Github GraphQL: 
![image](https://github.com/phrase/vulnerability_alerts_github_action/assets/1508/a3dc5aff-d210-42f2-a0e5-b04e6ee64a04)